code/lib/preview-api/src/modules/store/csf/composeConfigs.ts

```diff
@@ -42,6 +42,10 @@ export function composeConfigs<TRenderer extends Renderer>(
 ): ProjectAnnotations<TRenderer> {
   const allArgTypeEnhancers = getArrayField(moduleExportList, 'argTypesEnhancers');
   const stepRunners = getField(moduleExportList, 'runStep');
+  const initialGlobals = combineParameters(
+    getObjectField(moduleExportList, 'globals'),
+    getObjectField(moduleExportList, 'initialGlobals')
+  );
```

Comment:
Shouldn't all configs be normalized before reaching `composeConfigs`?

Reply:
Good call